tools/modules/software/module_postgres.sh

```
@@ -0,0 +1,103 @@
+	["module_postgres,doc_link"]="https://www.postgresql.org/docs/"
+	["module_postgres,group"]="Database"
+	["module_postgres,port"]="5432"
+	["module_postgres,arch"]="x86-64 arm64"
+)
+
+#
+# Module postgres
+#
+function module_postgres () {
+	local title="postgres"
+	local condition=$(which "$title" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] condition appears unused. Verify use (or export if used externally).

(SC2034)